src/mongo_client.ts

```diff
@@ -431,13 +431,14 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> implements
       | 'extendedMetadata'
     >;
 
+  private driverInfos: DriverInfo[] = [];
```

Comment:
nit: feel free to ignore if you feel strongly otherwise, but might this be better as something like `driverInfoList`? `driverInfos` reads a little too much like a typo of `driverInfo`, so it's harder to take in the difference at a glance

Reply:
My preference is to leave as-is because the type makes it clear, but if you'd prefer I can find a different name.